debian-setup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
+# MUST BE RUN WITH ROOT
+
+# Show all commands and exit upon failure
+set -eux
+
+# Make sure that all packages are up to date
+apt-get update
+apt-get upgrade -y
+
+# Install the official Debian packages that we need
+# curl and wget are not installed by default so this
+# is separate from the Clang/lld installation below
+# because we would need to at least install curl to
+# get LLVM's apt key
+apt-get install -y bc \
+                   binutils \
+                   binutils-aarch64-linux-gnu \
+                   binutils-arm-linux-gnueabi \
+                   bison \
+                   ccache \
+                   curl \
+                   flex \
+                   expect \
```

Comment:
Please sort these (flex vs expect).

If you're using vim, you can do this very quickly:
ctrl+v,j<repeat to highlight lines; maybe a better movement exists>, :sort, enter

Reply:
Oops, yes will do.